pkg/shell/base_index.js

```diff
@@ -33,6 +33,9 @@ function component_checksum(machine, component) {
 
 function Frames(index) {
     var self = this;
+    let language = document.cookie.replace(/(?:(?:^|.*;\s*)CockpitLang\s*=\s*([^;]*).*$)|^.*$/, "$1");
```

Comment:
This looks rather hairy -- isn't `cockpit.language` good enough for this? Or does it really have to be the "ll-CC" format?

Reply:
It is the same what we use a bit lower and the same we use in the shell's `<html>`. But I could not find any reference which would say if one or another is better. So I guess `cockpit.language` would be enough. @garrett any option about it?

Reply:
Indeed, I wasn't aware that this is already existing code. Thanks!